18830.xml

```diff
@@ -0,0 +1,173 @@
+                <Description><![CDATA[The Clean Session value to declare in the MQTT CONNECT message.]]></Description>
+            </Item>
+            <Item ID="3">
+                <Name>Keep Alive</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Mandatory</Mandatory>
+                <Type>Unsigned Integer</Type>
+                <RangeEnumeration>0..65535</RangeEnumeration>
+                <Units>s</Units>
+                <Description><![CDATA[The Keep Alive value to declare in the MQTT CONNECT message.]]></Description>
+            </Item>
+            <Item ID="4">
+                <Name>User Name</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>String</Type>
+                <RangeEnumeration></RangeEnumeration>
+                <Units></Units>
+                <Description><![CDATA[The User Name to declare in the MQTT CONNECT message.]]></Description>
+            </Item>
+            <Item ID="5">
+                <Name>Password</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Opaque</Type>
+                <RangeEnumeration>0..65535</RangeEnumeration>
+                <Units></Units>
+                <Description><![CDATA[The Password value to declare in the MQTT CONNECT message.]]></Description>
+            </Item>
+            <Item ID="6">
+                <Name>Security Mode</Name>
+                <Operations>RW</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Mandatory</Mandatory>
+                <Type>Integer</Type>
+                <RangeEnumeration>0..4</RangeEnumeration>
+                <Units></Units>
+                <Description><![CDATA[Determines which security mode is used
+                    0: PreShared Key mode
+                    1: Raw Public Key mode
+                    2: Certificate mode
+                    3: NoSec mode
+                    4: Certificate mode with EST
+                ]]></Description>
+            </Item>
+            <Item ID="7">
+                <Name>Public Key or Identity</Name>
+                <Operations>W</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Opaque</Type>
+                <RangeEnumeration></RangeEnumeration>
+                <Units></Units>
+                <Description><![CDATA[Stores the Device’s certificate, public key (RPK mode) or PSK Identity (PSK mode).]]></Description>
+            </Item>
+            <Item ID="8">
+                <Name>MQTT Broker Public Key</Name>
+                <Operations>W</Operations>
```

Comment:
This could be readable too?

Reply:
Indeed. This is modified in the new commit.